phases/phase-1/challenges/referral/saiSunkari19

```diff
@@ -0,0 +1,2 @@
+saiSunkari19
```

Comment:
Currently you cannot refer yourself. Please remove this in order to make this referral list valid 😁

Reply:
Oh, okay. Removed.

Reply:
Oh, okay. Removed.